src/cider/type/plan/UnaryExpr.cpp

```diff
@@ -162,6 +162,10 @@ JITExprValue& UOper::codegenCast(CodegenContext& context, Analyzer::Expr* operan
   } else {
     // input is primitive type
     FixSizeJITExprValue operand_val(operand->codegen(context));
```

Comment:
comments is misleading, FixSizeValue included all numeric type, update it in latest pr. https://github.com/intel/BDTK/pull/290